src/crypto/pubkeyEncrypt.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { assert } from "chai";
+import {loadTestWallet, TestWallet} from "../testutils/wallets";
```

Comment:
Formatting of imports (e.g. spaces vs. missing spaces).

Reply:
done